tests/pytest/conftest.py

```diff
@@ -3,3 +3,9 @@ def pytest_addoption(parser):
     parser.addoption(
         "--refdir", action="store", default="reference/docker", help="directory for reference files"
     )
+    parser.addoption(
+        "--image_tag", action="store", default="default", help="The docker image tag to run"
+    )
```

Comment:
Could you change this to specify the full image name instead of just the tag? I think it would be useful to be able to apply this to other images (ML is planning to make images based off of fv3gfs-python).

Reply:
Actually, feel free to leave this as-is and I'll adjust it when making the default behavior run both images.